boards/arm/mimxrt1170_evk/board.cmake

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2021, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(CONFIG_SOC_MIMXRT1176_CM7)
+board_runner_args(pyocd "--target=mimxrt1170_cm7")
+board_runner_args(jlink "--device=MIMXRT1176xxxA_M7" "--reset-after-load")
+endif()
+
+
+
+if(CONFIG_SOC_MIMXRT1176_CM4)
+board_runner_args(pyocd "--target=mimxrt1170_cm4")
```

Comment:
refs to jlinkscript dont make sense since that got dropped.

Reply:
Thank you for pointing this. I have removed this section.